### ISSUE TITLE
Clean up vacuole terms

### ISSUE THREAD

**Issue by anna-anagnostop** [mgijax/mammalian-phenotype-ontology#3933]
Amend the primary labels and definitions of the following vacuole terms to make them consistent. Use "fluid-filled vesicles" in their definition.

[brain vacuoles](https://www.informatics.jax.org/vocab/mp_ontology/MP:0008025)
[vacuolated lens](https://www.informatics.jax.org/vocab/mp_ontology/MP:0011697) --> lens vauoles
[podocyte vacuolization](https://www.informatics.jax.org/vocab/mp_ontology/MP:0011899) --> podocyte vacuoles
